### ISSUE TITLE
Not getting stat_activity_realtime

### ISSUE THREAD

**Issue by darshansh** [cybertec-postgresql/pgwatch2#271]
Hello, 
I have installed pgwatch2 1.7.1 version. I have set one server for monitoring. In Helthcehck  host is commining and metric values are also captured. But when I click on Stat_activity_realtime dashboard it is showing nothing. I have configured metric in Preset configs. What I missed ? 

![image](https://user-images.githubusercontent.com/14271741/80714341-6d70c280-8b12-11ea-9e76-e73648081dee.png)

![image](https://user-images.githubusercontent.com/14271741/80714389-7feafc00-8b12-11ea-915f-2c76f0d5503d.png)

**Comment by kmoppel** [comment]
Could you please also paste the configs info? And which image are you using - Influx or PG? Works for me for both with no problems...

**Comment by darshansh** [comment]
Thanks Kmoppel for quick response. 

On last Thursday I tried many things but not worked. It's strange. I restarted my VM  today and now it is working. Still one thing I observed is , if no connection are there in database still stat_activity_realtime showing all old captures. It should not show actually. 

![image](https://user-images.githubusercontent.com/14271741/80961486-ae294e00-8e28-11ea-9fc9-81f17a114168.png)
